authentik/stages/redirect/models.py

```diff
@@ -0,0 +1,49 @@
+"""authentik redirect stage"""
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.views import View
+from rest_framework.serializers import BaseSerializer
+
+from authentik.flows.models import Flow, Stage
+
+
+class RedirectMode(models.TextChoices):
+    """Mode a Redirect stage can operate in"""
+
+    STATIC = "static"
+    FLOW = "flow"
+
+
+class RedirectStage(Stage):
+    """Redirect the user to another flow, potentially with all gathered context"""
+
+    keep_context = models.BooleanField(default=True)
+    mode = models.TextField(choices=RedirectMode.choices)
```

Comment:
nit: do we even bother saving the mode if we can determine which to redirect as only one of the `target_` variables can be set via API?

Reply:
This is one of those forever-questions, right? We have 2 subtypes, how do we represent them and how do we store them in a relational database?

I chose Single Table Inheritance because `RedirectStage` is simple enough. (At least for now. Dear developer reading this in 2027, feel free to change this decision if you think it's gotten out of hand).